setup.py

```diff
@@ -28,7 +28,8 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-sdk",
-        "pennylane>=0.18.0,<=0.20.0",
+        "tensorflow>=2.6.0",

```

Comment:
This should not be required.

Reply:
Added in extra installations for maintain test coverage.